pkg/api/converterfromosaapi.go

```diff
@@ -51,7 +51,8 @@ func ConvertVLabsOpenShiftClusterToContainerService(oc *v1.OpenShiftCluster) *Co
 			}
 		}
 
-		cs.Properties.AgentPoolProfiles = make([]*AgentPoolProfile, len(oc.Properties.AgentPoolProfiles))
+		// +1 because master pool profile becomes agent pool profile
```

Comment:
So internally we have a masterProfile and we store it also as an agentPoolProfile? Why?

Reply:
> So internally we have a masterProfile and we store it also as an agentPoolProfile? Why?

I think I just need to remove that, it has confused everyone so far. :+1:  Will ping you to discuss options

Reply:
previous discussion on the topic https://github.com/openshift/openshift-azure/pull/172#discussion_r208959950